From e8ea57fc34667301394a133a654c7f1daa189a98 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 14 Oct 2018 09:49:14 +0300
Subject: [PATCH] Makefile: release: use target specific tag pointer

We have a generic release tag pointer for various jenkins build
jobs needs and it is called 'latest-embedded-release'.
Since we would like to differentiate between the targets, switch to
use target specific tag as a latest release pointer.
Also, slightly rename it, so the new tag name is:
<target>-latest-emb-release

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 1b294d2..30cad10 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,6 @@
 
 RELEASE_REPOS := 3rd-parties armcp btl buildroot buildroot-external \
 	fw_loader linux preboot specs specs_external tbot u-boot
-RELEASE_TAG_POINTER := latest-embedded-release
 RELEASE_BRANCH := $(shell \
 	git branch -r --points-at HEAD | \
 	grep -oE "origin/v[0-9]+\.[0-9]+\.[0-9]+" | \
@@ -110,6 +109,7 @@
 
 .PHONY: gaudi-release goya-release
 gaudi-release goya-release: hl-check-release hl-check-branch
+	$(eval RELEASE_TAG_POINTER=$(patsubst %-release,%,$@)-latest-emb-release)
 	$(eval EMBD_FULL_VER=hl-$(patsubst %-release,%,$@)-$(RELEASE_VER_COMPAT)-fw-$(EMBD_RELEASE_VER))
 	$(eval RELEASE_JOB_PROD_NAME="product_name=$(patsubst %-release,%,$@)")
 	$(eval RELEASE_JOB_TAG_PTR="embedded_release_tag=$(EMBD_FULL_VER)")
